<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgPreference</class>
 <widget class="QDialog" name="dlgPreference">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>767</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preference</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="documentMode">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Misc</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_10">
       <item row="0" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Network</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_9">
          <item row="2" column="0" colspan="2">
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>The way to get the download link for Kext upgrade</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="0" column="0">
              <widget class="QRadioButton" name="rbtnAPI">
               <property name="text">
                <string>Accessing the GitHub API (There is a limit to the number of visits per hour)</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QRadioButton" name="rbtnWeb">
               <property name="text">
                <string>Crawl web source code (No limit on the number of visits)</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QCheckBox" name="chkProxy">
                 <property name="text">
                  <string>HTTP &amp;&amp; HTTPS Proxy</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Host Name</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="txtHostName">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>127.0.0.1</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Port</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="txtPort">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>38457</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QGroupBox" name="groupBox_6">
            <property name="title">
             <string>Select the mirror server for upgrade download</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QComboBox" name="comboBoxNet">
               <property name="editable">
                <bool>true</bool>
               </property>
               <item>
                <property name="text">
                 <string>https://github.com/</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>https://download.fastgit.org/</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>https://ghproxy.com/https://github.com/</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>https://gh.api.99988866.xyz/https://github.com/</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>https://gh.msx.workers.dev/https://github.com/</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>https://gh.xiu2.xyz/https://github.com/</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>https://git.yumenaka.net/https://github.com/</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>https://pd.zwc365.com/seturl/https://github.com/</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QToolButton" name="btnTest">
               <property name="text">
                <string>Testing</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QGroupBox" name="groupBox_7">
            <property name="title">
             <string>Select the mirror server for web access</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_6">
             <item row="0" column="0">
              <widget class="QComboBox" name="comboBoxWeb">
               <property name="editable">
                <bool>true</bool>
               </property>
               <item>
                <property name="text">
                 <string>https://github.com/</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>https://hub.fastgit.org/</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QToolButton" name="btnPing">
               <property name="text">
                <string>Testing</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>UI Appearance (effective after the next app opening)</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="1" column="0">
           <widget class="QGroupBox" name="groupBox_4">
            <property name="title">
             <string>Toolbar button</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_7">
             <item row="0" column="0">
              <widget class="QCheckBox" name="chkRecentOpen">
               <property name="text">
                <string>Recently Open</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QCheckBox" name="chkOpenDir">
               <property name="text">
                <string>Browse Catalog</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QCheckBox" name="chkMountESP">
               <property name="text">
                <string>Mount ESP</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QCheckBox" name="chkBackupEFI">
               <property name="text">
                <string>Backup EFI</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QCheckBox" name="chkDatabase">
               <property name="text">
                <string>Database</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="chkHideToolbar">
            <property name="text">
             <string>Hide Toolbar</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Additional Options</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="1" column="0">
           <widget class="QCheckBox" name="chkShowVolName">
            <property name="text">
             <string>Show all volume names when mounting ESP</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="chkBoxLastFile">
            <property name="text">
             <string>Automatically open the last opened file</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="chkSmartKey">
            <property name="text">
             <string>Smart adaptation to OpenCore upgrades.</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Kext Upgrade URL</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="3" column="0">
        <widget class="QToolButton" name="btnAdd">
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="QToolButton" name="btnStop">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="6">
        <widget class="QTextEdit" name="textEdit">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'SimSun'; font-size:9pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="3" column="5">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>458</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0" colspan="6">
        <widget class="QTableWidget" name="tableKextUrl">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <column>
          <property name="text">
           <string>Kext Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Source URL</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QToolButton" name="btnOpenUrl">
         <property name="text">
          <string>Open URL in browser</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QToolButton" name="btnDel">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QToolButton" name="btnDownloadKexts">
         <property name="text">
          <string>Download Kexts to the user database</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="6">
        <widget class="QLineEdit" name="myeditFind">
         <property name="placeholderText">
          <string>Find</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
